spec/controllers/conversations_controller_spec.rb

```diff
@@ -0,0 +1,160 @@
+require 'rails_helper'
+
+RSpec.describe ConversationsController do
+  let(:valid_session) { {} }
```

Comment:
Do you really need to initialize this here?

Reply:
Not really, thanks!